src/Toml/Printer.hs

```diff
@@ -20,11 +22,21 @@ import qualified Data.HashMap.Strict as HashMap
 import qualified Data.List.NonEmpty as NonEmpty
 import qualified Data.Text as Text
 
+{- | Configures the pretty printer. -}
+data PrintOptions = PrintOptions
+    { shouldSort :: Bool  -- ^ should table keys be sorted ot shouldn't
+    , indent     :: Int   -- ^ indentation size
+    } deriving (Show)
+
+{- | Default printing options. -}
+defaultOptions :: PrintOptions
+defaultOptions = PrintOptions True 2
+
 -- Tab is equal to 2 spaces now.
 tab :: Int -> Text
 tab n = Text.cons '\n' (Text.replicate (2*n) " ")
```

Comment:
I think you should also patch this function and replace `2 * n` with `n`. As I understand correctly, the `tab` function will be given numbers `0, 2, 4, 6, 8, ...` or `0, 4, 8, 12, ...` (depending on the indentation size) and previously it took numbers `0, 1, 2, 3, ...`. So instead of multiplying sequent natural numbers by indentation we can just add indent to the `nextI` (as you already did) but also remove multiplier from the `tab` function